resources/assets/components/pages/ShowSubmissionPage/ShowSubmissionPage.js

```diff
@@ -35,7 +36,7 @@ const ShowSubmissionPage = ({ match }) => {
     },
   });
 
-  const postImageUrl = get(postData, 'post.url', null);
+  const { url: postImageUrl, campaignId } = get(postData, 'post') || {};
```

Comment:
Could this use the same default value?
```suggestion
  const { url: postImageUrl, campaignId } = get(postData, 'post', {});
```

Reply:
I experienced the same issue here per https://github.com/DoSomething/phoenix-next/pull/2456#discussion_r538770345